.github/workflows/continuous-integration-build.yml

```diff
@@ -65,16 +65,16 @@ jobs:
         run: |
           echo 'assets/images/templates' >> .distignore
           npm run build:js
-          npm run workflow:build-plugin -- --cdn
-          npm run workflow:bundle-plugin web-stories-composer.zip -- --composer --skip-delete
-          npm run workflow:bundle-plugin web-stories.zip -- --skip-delete
+          npm run workflow:version -- --nightly
```

Comment:
What does change this to nightly mean?

Reply:
That the current commit / revision is added to the plugin version in the main plugin file.